docs/changelog/1_0_a7.rst

```diff
@@ -0,0 +1,248 @@
+        "proposed": {
+          "confidence": 0.9956623333333332,
+          "operation_id": "ALTER TYPE default::Foo",
+          "prompt": "did you alter object type 'default::Foo'?",
+          "statements": [{"text": "ALTER TYPE default::Foo {\n
+          ALTER PROPERTY val {\n        DROP default;\n    };\n};"}]
+        }
+      }
+    ]
+
+The proposed statements will ``DROP default`` for our property, so all
+seems to be in order and we can apply this migration, too, using
+:eql:stmt:`POPULATE MIGRATION` and :eql:stmt:`COMMIT MIGRATION`.
+
+We're currently working on a CLI tool for managing migrations more
+gracefully and without the need for the user to rely on these
+low-level commands (like ``START MIGRATION`` or ``DESCRIBE CURRENT
+MIGRATION AS JSON``). The migration tool is going to use these
+commands behind the scenes, though.
+
+We've also made improvements to the following migration features:
+
+* Better overall dependency tracking to make sure that migration to
+  the new state can be resolved and produces valid command sequence.
+* Type, index and alias renaming while keeping track of affected
+  expressions to make sure they don't become invalid (:eql:gh:`#1841`)
+* Function renaming (:eql:gh:`#1971`)
+* Moving a type between modules (:eql:gh:`#1890`).
+* Changing base types and changing where constraints are defined
+  (:eql:gh:`#1996`).
+
+
+Command-Line Tools
+==================
+
+* Default user and default database are now simply ``edgedb`` and no
+  longer named after the system user.
+* Add ``--connect-timeout`` to control how long to wait for EdgeDB
+  response (`#191 <https://github.com/edgedb/edgedb-cli/pull/191>`_).
+* Add ``--dsn`` as a connection option (`#176
+  <https://github.com/edgedb/edgedb-cli/issues/176>`_).
+* Add ``migration-log`` command to view applied migrations (`#200
+  <https://github.com/edgedb/edgedb-cli/pull/200>`_).
+* Non-interactive error messages are prefixed by ``edgedb error:
+  ...``, to quickly spot which tool has errored in scripts.
+* Improve accuracy of syntax error reporting in REPL (:eql:gh:`#1959`).
+* REPL now supports full range of datetime values (`#192
+  <https://github.com/edgedb/edgedb-cli/pull/192>`_).
+* ``\lt`` in REPL doesn't show implicit internal types (unions and
+  intersections) (`#169 <https://github.com/edgedb/edgedb-cli/issues/169>`_).
+* Remove ``\set introspect-types`` in REPL and show typenames by
+  default.
+
+
+Server Command-Line
+===================
+
+* Make ``server install`` friendlier on linuxes without systemd
+  allowing foreground run (`#171
+  <https://github.com/edgedb/edgedb-cli/pull/171>`_).
+* When installing server ``DEBIAN_FRONTEND`` is now ``noninteractive`` by
+  default and is overridable (`#188
+  <https://github.com/edgedb/edgedb-cli/pull/188>`_).
+* Add ``edgedb server logs`` (`#172
+  <https://github.com/edgedb/edgedb-cli/pull/172>`_).
+* Add ``server info`` command.
+* Add shutdown timeout of 1 second.
```

Comment:
This is super-unclear. Better:
* When exiting from the REPL with Ctrl+D client now waits max of 1 second for server response

But generally, if you skipped all bugfixes anyway, this might go unnoticed too.

Reply:
Perhaps I could just drop it from the highlights.